deploy/docker/scripts/run-java.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# Ref -Dlog4j2.formatMsgNoLookups=true https://spring.io/blog/2021/12/10/log4j2-vulnerability-and-spring-boot
+java_cmd="java ${APPSMITH_JAVA_ARGS:-} -Dserver.port=8082 -Djava.security.egd=file:/dev/./urandom -Dlog4j2.formatMsgNoLookups=true -jar server.jar"
+exec $java_cmd
```

Comment:
Is there a reason we are sticking this in a variable and then passing to `exec`, instead of just:

```suggestion
exec java ${APPSMITH_JAVA_ARGS:-} -Dserver.port=8082 -Djava.security.egd=file:/dev/./urandom -Dlog4j2.formatMsgNoLookups=true -jar server.jar
```